docs/docsite/rst/userguide/inventories.rst

```diff
@@ -1120,6 +1120,46 @@ This inventory source uses the `terraform_state <https://github.com/ansible-coll
 6. To add hosts for AWS EC2, GCE, and Azure instances, the Terraform state file in the backend must contain state for resources already deployed to EC2, GCE, or Azure. Refer to each of the Terraform providers' respective documentation to provision instances.
 
 
+.. _ug_source_ocpv:
+
+
+OpenShift Virtualization
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. index::
+   pair: inventories; OpenShift
+   pair: inventories; OCP
+   pair: inventory source; OpenShift virtualization
+
+
+This inventory source uses a cluster that is able to deploy OpenShift (OCP) virtualization. In order to configure an OCP virtualization requires a virtual machine deployed in a specific namespace and an OpenShift or Kubernetes API Bearer Token credential.
+
+1. To configure this type of sourced inventory, select **OpenShift Virtualization** from the Source field.
+2. The Create new source window expands with the required **Credential** field. Choose from an existing Kubernetes API Bearer Token credential. For more information, refer to :ref:`ug_credentials_ocp_k8s`. In this example, the ``cmv2.engineering.redhat.com`` credential is used.
+
+3. You can optionally specify the verbosity, host filter, enabled variable/value, and update options as described in the main procedure for :ref:`adding a source <ug_add_inv_common_fields>`. 
+
+4. Use the **Source Variables** field to override variables used by the ``controller`` inventory plugin. Enter variables using either JSON or YAML syntax. Use the radio button to toggle between the two. For more information on these variables, see the `Kubernetes (K8s) inventory source <https://github.com/ansible-collections/kubernetes.core/blob/main/docs/kubernetes.core.k8s_inventory.rst>`_ documentation for detail.
+
+  In the example below, the ``connections`` variable is used to specify access to a particular namespace in a cluster. 
+
+  ::
+
+    ---
+    connections:
+    - namespaces:
+      - hao-test
+
+
+  .. image:: ../common/images/inventories-create-source-ocpvirt-example.png
+
```

Comment:
lets reference https://github.com/ansible-collections/kubernetes.core/blob/main/docs/kubernetes.core.k8s_inventory.rst#parameters

Reply:
similiar to what gcp inventory plugin 
```
Use the Source Variables field to override variables used by the gcp_compute inventory plugin. Enter variables using either JSON or YAML syntax. Use the radio button to toggle between the two. For a detailed description of these variables, view the [gcp_compute inventory plugin documentation](https://cloud.redhat.com/ansible/automation-hub/repo/published/google/cloud/content/inventory/gcp_compute).
```

Reply:
So for this, it needs to say `Use the Source Variables field to override variables used by the gcp_compute inventory plugin`?

Reply:
Or does it need to say `Use the Source Variables field to override variables used by the kubernetes.core.k8s inventory plugin`?